architecture/adr-014-module-catalog.md

```diff
@@ -0,0 +1,28 @@
+<!-- Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. -->
+
+| Type          | ID            | Status        | Title                   |
+| ------------- | ------------- | ------------- | ----------------------- |
+| ADR           | adr-014       | Accepted      | Multiple ModuleCatalogs |
```

Comment:
[ZKRobi](https://github.com/ZKRobi) [4 days ago](https://github.com/morganstanley/ComposeUI/pull/569#discussion_r1555945826)
I'd suggest discussing and putting the ADR to Accepted state before merging it together with the PR

[lilla28](https://github.com/lilla28) [3 days ago](https://github.com/morganstanley/ComposeUI/pull/569#discussion_r1557297627)
I will create a new commit with the change of the status once everything is resolved and we discussed.